### ISSUE TITLE
Single-Selection Matrix - A screen reader doesn't read row captions

### ISSUE THREAD

**Issue by JaneSjs** [surveyjs/survey-library#7273]
User Issue: T15336 - Matrix question types and screen reader
https://surveyjs.answerdesk.io/internal/ticket/details/T15336
***
The NVDA screen reader doesn't announce row captions/titles when tabbing through matrix elements. Therefore, it is hard to understand which row is being answered. This issue is reproducible in [our demo](https://surveyjs.io/form-library/examples/single-selection-matrix-table-question/reactjs).
